portfolio/src/main/webapp/contact.css

```diff
@@ -0,0 +1,83 @@
+html, body {
+    width: 100%;
+    height: 100%;
+    margin: 0;
+}
+
+#header {
+    display: flex;
+    flex-direction: row;
+    padding: 10px;
+    align-items: center;
+    background-color: rgb(192, 192, 192);
+    background-image: linear-gradient(rgb(192, 192, 192), rgb(129, 129, 129));
+}
+
+#header img {
+    height: 75px;
+    width: 100px;
+}
+
+.contact_content {
+    min-height: 100%;
+    display: flex;
+    flex-direction: column;
+    align-items: stretch;
+    background-color: black;
+}
+
+#message {
+    flex-grow: 1;
+}
+
+#message, #footer {
+    flex-shrink: 0;
+}
+
+#message {
+    align-content: center;
+    background-color: yellow;
+    width: 100%;
+}
```

Comment:
Remember to remove this unused message stuff from this pull request as well.

Hopefully in the future you won't have to update things like this in multiple PRs since you'll get into the habit of creating the different branches for the different PRs off of master instead of off of each other =)